### PR TITLE
ODH-1707 UsageCollectorReports Dropped Events

### PR DESCRIPTION
**Pull request by BenjaminDebotte**
My previous PR was a complete mess. Here is a new version with fixes and tests as requested

### CODE REVIEW

**Comment by vunder-kind** [comment]
Benjamin, we have no field @timestamp in our input file, change it to "timestamp" and it should be a epoch format. So in your schema you should use StringType for timestamp and then use epoch_to_iso converter. Look to already implemented drivers. Also you have no import for CompoundAggregation. Please merge master to your currrent branch and move you files to eos_stb_reports paths like all others. Ask Ali please if you have any problems

File: src/test/it/applications/basic_analytics/eos_stb_reports/test_eos_stb_report_usagecollector_droppedevents.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from applications.basic_analytics.eos_stb.general_driver import GeneralEosStbProcessor
+from test.it.core.base_spark_test_case import BaseSparkProcessorTestCase
+
+
+class EosStbAnalyticsTestCase(BaseSparkProcessorTestCase):
+    def test_events(self):
+        self._test_pipeline(
+            configuration_path="test/it/resources/basic_analytics/eos_stb/general/configuration.yml",
+            processor_creator=GeneralEosStbProcessor.create_processor,
```
Comment (alimaken) [inline src/test/it/applications/basic_analytics/eos_stb_reports/test_eos_stb_report_usagecollector_droppedevents.py]: You are not using GeneralEosStbProcessor. So the import is wrong.

File: src/test/it/applications/basic_analytics/eos_stb_reports/test_eos_stb_report_usagecollector_droppedevents.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+            input_dir="test/it/resources/basic_analytics/eos_stb/general/input",
```
Comment (alimaken) [inline src/test/it/applications/basic_analytics/eos_stb_reports/test_eos_stb_report_usagecollector_droppedevents.py]: Probably not correct for the same reason as in previous comment.

File: src/test/it/applications/basic_analytics/eos_stb_reports/test_eos_stb_report_usagecollector_droppedevents.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+            expected_result_file="test/it/resources/basic_analytics/eos_stb/general/expected_result.txt"
+        )
```
Comment (alimaken) [inline src/test/it/applications/basic_analytics/eos_stb_reports/test_eos_stb_report_usagecollector_droppedevents.py]: Probably not correct for the same reason as in previous comment.

**Pull request closed without merging**